service/src/main/kotlin/xyz/lebkuchenfm/external/storage/dropbox/models/ShareFile.kt

```diff
@@ -0,0 +1,22 @@
+package xyz.lebkuchenfm.external.storage.dropbox.models
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ShareFileSettings(
```

Comment:
I would suggest to rename all of the data classes that we use to communicate with Dropbox with a `DropboxClassName` prefix.

```suggestion
data class DropboxShareFileSettings(
```

Reply:
those are already in dropbox package but ok